app/stac_api/views.py

```diff
@@ -181,8 +181,8 @@ def get_queryset(self):
 
     def list(self, request, *args, **kwargs):
         if not Collection.objects.filter(name=self.kwargs['collection_name']).exists():
-            logger.error("The collection %s does not exist", self.kwargs['collection_name'])
-            raise Http404("This collection does not exists.")
+            logger.error("The collection, %s, does not exist", self.kwargs['collection_name'])
+            raise Http404(f"This collection, {self.kwargs['collection_name']}, does not exists.")
```

Comment:
`The collection {self.kwargs['collection_name']} does not exists.` seems to me more appropriate.